src/website/views/components/accordion/_examples.pug

```diff
@@ -171,6 +171,122 @@ h3 Portal
         </div>
       </div>
```

Comment:
@janibolkvadze same as the previous comment about missing the js script that initializes the accordion in the copy section